sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -536,6 +536,25 @@ object functions {
   def min(columnName: String): Column = min(Column(columnName))
 
   /**
+   * Aggregate function: returns the exact percentile(s) of the expression in a group at pc with
+   * range in [0, 1].
+   *
+   * @group agg_funcs
+   * @since 2.1.0
+   */
+  def percentile(e: Column, pc: Seq[Double]): Column =
+    withAggregateFunction {Percentile(e.expr, pc)}
```

Comment:
Style: match the other code, e.g.: `def min(e: Column): Column = withAggregateFunction { Min(e.expr) }`